### PR TITLE
Take users out of workspaces

### PR DESCRIPTION
**Pull request by timmillwood**

### CODE REVIEW

File: multiversion.module [hunk 1]
```diff
@@ -122,14 +123,16 @@ function multiversion_entity_base_field_info(EntityTypeInterface $entity_type) {
...
     // then we don't need to join the data table (which it isn't by default).
-    $fields['workspace'] = BaseFieldDefinition::create('workspace_reference')
-      ->setLabel(t('Workspace reference'))
-      ->setDescription(t('The workspace this entity belongs to.'))
-      ->setSetting('target_type', 'workspace')
-      ->setRevisionable(FALSE)
-      ->setTranslatable(FALSE)
-      ->setCardinality(1)
-      ->setReadOnly(TRUE);
+    if ($entity_type->id() != 'user') {
```
Comment (dickolsson) [inline multiversion.module]: Should we not check for the annotation `workspace = FALSE` here instead? I don't like hard-coding users in here.

File: src/Workspace/ConflictTracker.php [hunk 2]
```diff
@@ -15,7 +16,7 @@ class ConflictTracker implements ConflictTrackerInterface {
 
   /**
    * The id of current workspace to query.
-   *
+   *y
```
Comment (dickolsson) [inline src/Workspace/ConflictTracker.php]: Typo

Comment (timmillwood) [inline src/Workspace/ConflictTracker.php]: Bloody kids banging on my keyboard!!!

File: src/Entity/Index/SequenceIndex.php [hunk 3]
```diff
@@ -57,29 +57,44 @@ public function useWorkspace($id) {
...
   }
 
   /**
    * {@inheritdoc}
    */
   public function getRange($start, $stop = NULL, $inclusive = TRUE) {
-    return $this->sortedSetStore()->getRange($start, $stop, $inclusive);
+    $range = $this->sortedSetStore()->getRange($start, $stop, $inclusive);
+    if (empty($range)) {
+
```
Comment (jeqq) [inline src/Entity/Index/SequenceIndex.php]: This if is empty

**Pull request merged**